app/src/main/java/org/schabi/newpipe/util/ThemeHelper.java

```diff
@@ -231,4 +269,28 @@ private static String getSelectedThemeString(final Context context) {
         return PreferenceManager.getDefaultSharedPreferences(context)
                 .getString(themeKey, defaultTheme);
     }
+
+    /**
+     * Get the device theme
+     * <p>
+     * It will return true if the device's theme is dark, false otherwise.
+     * <p>
+     * From https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#java
+     *
+     * @param context the context to use
+     * @return true: dark theme, false: light or unknown
+     */
+    private static boolean getDeviceTheme(final Context context) {
```

Comment:
The name of this method is irritating. What about
```suggestion
    private static boolean isDeviceNightModeEnabled(final Context context) {
```

Reply:
isDeviceDarkThemeEnabled